src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -2963,30 +2966,52 @@
-    ("SveLoadMaskedUnOpTest.template",    new Dictionary<string, string> {["TestName"] = "SveLoadVectorUInt32ZeroExtendToInt64",                                                                   ["Isa"] = "Sve",                                   ["Method"] = "LoadVectorUInt32ZeroExtendToInt64",                                    ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Int64",   ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Int64",                                   ["Op2BaseType"] = "UInt32",                                                              ["LargestVectorSize"] = "8",                                                                ["NextValueOp2"] = "TestLibrary.Generator.GetUInt32()",                                                                                        ["ValidateIterResult"] = "firstOp[i] != result[i]"}),
-    ("SveLoadMaskedUnOpTest.template",    new Dictionary<string, string> {["TestName"] = "SveLoadVectorUInt32ZeroExtendToUInt64",                                                                  ["Isa"] = "Sve",                                   ["Method"] = "LoadVectorUInt32ZeroExtendToUInt64",                                   ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt64",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "UInt64",                                  ["Op2BaseType"] = "UInt32",                                                              ["LargestVectorSize"] = "8",                                                                ["NextValueOp2"] = "TestLibrary.Generator.GetUInt32()",                                                                                        ["ValidateIterResult"] = "firstOp[i] != result[i]"}),
+    ("SveLoadMaskedUnOpTest.template",    new Dictionary<string, string> { ["TestName"] = "SveLoadVectorByteZeroExtendToInt16",                                                                    ["Isa"] = "Sve",                                    ["Method"] = "LoadVectorByteZeroExtendToInt16",                                      ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Int16",   ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Int16",                                   ["Op2BaseType"] = "Byte",                                                                ["LargestVectorSize"] = "64",                                                               ["NextValueOp2"] = "TestLibrary.Generator.GetByte()",                                                                                          ["ValidateIterResult"] = "firstOp[i] != result[i]"}),
```

Comment:
LargestVectorSize is in bytes? So this is now 512bits. Largest possible SVE size in the spec is 2048bits.
Happy to leave it at 512 though

Reply:
yes, in case I need to push more updates on this PR, I will change it to 256 or will do it in next PR.